docs/en_US/network-dumps.md

```diff
@@ -0,0 +1,69 @@
+
+# Security Warning
+## Network dumps will oftentimes have sensitive information, including emails, passwords, usernames, IP addresses, etc. Because of this, ensure you only provide network dumps to trusted individuals. When submitting network dumps to Pretendo Network developers your dumps are uploaded and stored in a private channel. This channel can only be accessed by a small number of people, not even all developers have access.
+
+# Submissions
+There are multiple ways to submit network dumps. You may either contact a developer directly, or upload your network dumps to any channel in the Discord server. If uploaded to Discord, the Bandwidth bot will automatically detect the dumps and reupload them to our private channel, deleting the original message for security purposes.
+
+When submitting we ask that you do the following:
+
+1. Only submit dumps for one game at a time, to keep the files organized. Sending multiple dumps of multiple games makes it difficult to know which dumps go to which game,
+2. Perform only small actions if possible. Each session should be focussed on as few aspects of the game as possible to minimize "noise" (unrelated packets) which makes it difficult to decipher what is actually going on. For example, rather than uploading a session of playing random courses on Super Mario Maker it would be more helpful if the session was focussed on one part of the game, such as 100 Mario Challenge, starring a course, etc. That way it's clear exactly what the packets are doing in game.
+3. When submitting, please provide a detailed description of what was done in the session. For example "Loaded course world and played a random course, starred the course and played a recommended course". This will add further context as to what the packets are doing in the dumps.
+
+# Game Packets
+Game packet dumps are the packets sent during a session to a game server. These packets are typically encrypted, so additional data will need to be submitted in order for us to decrypt them. There are several ways to dump these packets and the data needed to decrypt them.
+
+### Wii U (HokakuCafe)
+The easiest way to dump game server packets from a Wii U is to use [HokakuCafe](https://github.com/PretendoNetwork/HokakuCafe). This is an Aroma setup module which patches IOSU to write all network frames to a `.pcap` file on the SD card.
+
+To use HokakuCafe download the [latest release](https://github.com/PretendoNetwork/HokakuCafe/releases/latest) setup module (`30_hokaku_cafe.rpx`). Place this file on the SD card at `wiiu/environments/[ENVIRONMENT]/modules/setup` and insert it back into the Wii U. Start your Wii U once, turn it off, and open the newly created `HokakuCafe/config.ini` file on the SD card. Set `Mode` to `ALL`. At the time of writing, HokakuCafe has a bug where it will not dump all packets unless `Mode` is set to `ALL`. Insert the SD card back into the Wii U and all network traffic flowing through your console should be written to a PCAP.
```

Comment:
> HokakuCafe has a bug

well I wouldn't say a "bug" but it's intended. By default it will only dump PRUDP packets, as that's what most Nintendo games use. Unless you mean the PRUDPv0 bug, in that case all packets should be captured with `UDP` mode too, but it doesn't hurt recommending setting the option to `ALL`.

Reply:
I am talking about the v0 bug yeah, I figured just telling people to use `ALL` would be easier than explaining the actual bug. I can change it to `UDP` though, since tbh if users are using proxies we don't need anything else